src/robot_model_builder.cpp

```diff
@@ -9,6 +9,10 @@
 
 namespace linear_feedback_controller {
 
+const Eigen::Index RobotModelBuilder::free_flyer_nq_ = 7;
+
+const Eigen::Index RobotModelBuilder::free_flyer_nv_ = 6;
+
```

Comment:
```suggestion
const Eigen::Index RobotModelBuilder::k_free_flyer_nq_ = 7;

const Eigen::Index RobotModelBuilder::k_free_flyer_nv_ = 6;

```
This complies with google coding style